.gitignore

```diff
@@ -42,3 +42,4 @@ web/static/js/dll/*
 
 npm-debug.log*
 .eslintcache
+browser_logs.log
```

Comment:
❤️ thanks for catching this! it didn't show up in my history, but i must have overlooked it!